docs/product/issues/issue-details/sentry-ai/index.mdx

```diff
@@ -63,6 +67,7 @@ The data used for these features includes:
 - Stack traces
 - Sentry spans
 - DOM interactions
+- Profiles
 - Relevant code from linked repositories
```

Comment:
commit history?

Reply:
- Relevant code from linked repositories
this covers it no?